### ISSUE TITLE
Scroll node - wrong instruction/description

### ISSUE THREAD

**Issue by dragonfly69** [HendrikRoth/node-red-contrib-oled#10]
Hi,
Availiable in oled package scroll node has got description from WiFi node, 
Is it possible to get any help how to use this scroll node ?
How to turn it On/Off, What modes are available up/down/right/left,
Does it allows to increase available space ?

**Comment by paddybeee** [comment]
I would agree with dragonfly69, also seems to do some weird things with your display if you add it, then remove it, i have had my other icons start scrolling once i removed it, reboot does not fix the issue,

**Comment by danprof** [comment]
Scrolls the display
	<p>Payload should contain direction, start position and stop position.</p>
	<p>direction: <b>msg.payload.direction</b></p>
	<p>start: <b>msg.payload.start</b></p>
 	<p>stop: <b>msg.payload.stop</b></p>
	<p>Example: <b>{"direction":"left","start":0,"stop":128}</b></p>
